Classes/Domain/Factory/Database.php

```diff
@@ -51,10 +52,11 @@ protected function getQueryBuilder(): QueryBuilder
             && ApplicationType::fromRequest($this->getServerRequest())->isFrontend()
         ) {
             $queryBuilder->setRestrictions(GeneralUtility::makeInstance(FrontendRestrictionContainer::class));
-            if ((GeneralUtility::makeInstance(Typo3Version::class))->getMajorVersion() < 12) {
+            if ((GeneralUtility::makeInstance(Typo3Version::class))->getMajorVersion() < 13) {
                 // do not use FrontendWorkspaceRestriction
                 $queryBuilder->getRestrictions()
                     ->removeByType(FrontendWorkspaceRestriction::class)
+                    ->removeByType(HiddenRestriction::class)
```

Comment:
now die hidden child in live workspace is also visible in FE (and it should not depend on TYPO3 Version), i will look for "a better" solution